app/ApiClients/GraylogApi.php

```diff
@@ -97,13 +97,14 @@ public function query(string $query = '*', int $range = 0, int $limit = 0, int $
      */
     public function buildSimpleQuery(?string $search = null, ?Device $device = null): string
     {
+        $field = Config::get('graylog.query.field');
```

Comment:
What if the user has not configured a default? Maybe this should use the configured option and fallback to the default if no configuration is present? Also, a small mention of this feature could be added to the documentation page ;)